src/utils/secretmanager.rs

```diff
@@ -0,0 +1,66 @@
+use async_trait::async_trait;
+use aws_config::{BehaviorVersion, Region};
+use aws_sdk_secretsmanager::Client as awsClient;
```

Comment:
follow the conventional naming for types
```suggestion
use aws_sdk_secretsmanager::Client as AwsClient;
```